typescript-light-bulb/main.ts

```diff
@@ -0,0 +1,18 @@
+const $bulb = document.querySelector('.bulb');
+const $background = document.querySelector('.background');
+let lightStatus = 'off';
+if (!$bulb) throw new Error('Someone had better hurry over to Home Depot!');
+$bulb.addEventListener('click', clickHandler);
```

Comment:
Rather than using a `string` here, what is another data type we could use instead that would make more sense for two options?